### PR TITLE
[ui] add new variable

### PR DESCRIPTION
**Pull request by neversaynever0502**
How to test:
1. "yarn start" to run the example
2. go into core/ui and "yarn start" to listen the ui realtime changes
3. change theme variable in core/ui/src/variable.less
4. the example ui will be changed in the meantime

### CODE REVIEW

File: core/ui/src/assets/IconChevronUp.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
+// import '../variable.less'
```
Comment (RustySol) [inline core/ui/src/assets/IconChevronUp.tsx:2]: let's remove this?

**Review by RustySol — APPROVED** [review]

File: core/ui/src/assets/IconChevronUp.tsx (hunk 1 above)
Comment (neversaynever0502) [reply core/ui/src/assets/IconChevronUp.tsx]: Yes, done.

File: core/ui/src/components/PoweredBy/style.less [hunk 2]
```diff
@@ -30,4 +32,6 @@
   svg {
     margin-left: 8px;
   }
+
+  --IconCandyShopOutlined: @candy-shop-color-primary;
```
Comment (haVincy) [inline core/ui/src/components/PoweredBy/style.less:36]: CSS variables tend to use lower cases.

Seems we will use the same color `@candy-shop-color-primary` for `IconCandyShop` & `IconCandyShopOutlined` & Chevron at this moment. 

We can just declare one css variable in root? 

```less
.root {
   --candy-shop-activate-icon-color: @candy-shop-color-primary
}
```

In various places for SVG:
```tsx
fill="var(--candy-shop-activate-icon-color)"
```

Comment (neversaynever0502) [reply core/ui/src/components/PoweredBy/style.less]:
```
:root {
   --candy-shop-activate-icon-color: @candy-shop-color-primary
}
```
added, worked fine.
also changed other css variable to use lower cases.

**Review by vanminh1701 — APPROVED** [review]

**Review by RustySol — APPROVED** [review]

**Pull request merged**